zoo/box2d/bipedalwalker/config/bipedalwalker_cont_sampled_unizero_config.py

```diff
@@ -42,17 +42,17 @@
             world_model_cfg=dict(
                 obs_type='vector',
                 num_unroll_steps=num_unroll_steps,
-                policy_entropy_loss_weight=1e-4,
+                policy_entropy_loss_weight=5e-3,
                 continuous_action_space=continuous_action_space,
                 num_of_sampled_actions=K,
                 sigma_type='conditioned',
```

Comment:
why still conditioned here

Reply:
We will rename it to policy_entropy_weight latter